### ISSUE TITLE
Talkback Support

### ISSUE THREAD

**Issue by Lily418** [Lily418/candle-light#45]
Version 1 had good Talkback support, this has regressed whilst implementing the new designs. I should fix this before making a new Play Store release.

**Comment by Lily418** [comment]
Blocked by this issue on React Native
https://github.com/facebook/react-native/issues/17624

Cannot interact with text input when voiceover is on, I'm following this issue and will reopen this when that is resolved
